pkg/objectstore/s3minio.go

```diff
@@ -135,7 +146,7 @@ func GetS3Client(ctx context.Context, objectStore *v1.S3ObjectStore, dynamicClie
 		log.Tracef("Found secretKey [%s] in secret [%s] in namespace [%s]", secretKey, secretName, secretNs)
 	}
 	// if no s3 credentials are provided, use IAM profile, this means passing empty access and secret keys to the SetS3Service call
-	s3Client, err := SetS3Service(objectStore, accessKey, secretKey, true)
+	s3Client, err := SetS3Service(objectStore, accessKey, secretKey, false)
```

Comment:
Need to inject an additional option when storing secrets to set SSL to false, but always safeguard to defaulting to true unless explicitly opted out of